src/connector/src/source/cdc/source/reader.rs

```diff
@@ -98,7 +97,9 @@ impl SplitReader for CdcSplitReader {
 impl CdcSplitReader {
     #[try_stream(boxed, ok = Vec<SourceMessage>, error = anyhow::Error)]
     async fn into_data_stream(self) {
-        let cdc_client = ConnectorClient::new(&self.conn_props.connector_node_addr).await?;
+        let cdc_client = self.source_ctx.connector_client.clone().ok_or(anyhow!(
```

Comment:
May use `ok_or_else(|| anyhow!(...))` to avoid creating an `anyhow` even nothing goes wrong.

Surprisingly, calling `anyhow` takes more than a second in my test.

Reply:
Good point.